katello/katello-backup

```diff
@@ -224,15 +270,31 @@ def offline_backup(database, dir_path = nil)
   end
 end
 
+def compress_files
+  psql = spawn('gzip', 'pgsql_data.tar') if @databases.include? "pgsql"
+  mongo = spawn('gzip', 'mongo_data.tar')
+  Process.wait(psql) if @databases.include? "pgsql"
+  Process.wait(mongo)
+end
+
+def plugin_list
+  if @is_foreman_proxy_content
+    eval(`curl -k https://$(hostname):9090/features`)
```

Comment:
@ehelms to return an array, I figured there was no user input being entered, so using `eval` would be ok

here is the output with and without eval
```
irb(main):001:0> eval(`curl -k https://$(hostname):9090/features`)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    58  100    58    0     0    349      0 --:--:-- --:--:-- --:--:--   349
=> ["logs", "pulpnode", "puppet", "puppetca", "templates", "tftp"]
irb(main):002:0> `curl -k https://$(hostname):9090/features`
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    58  100    58    0     0    369      0 --:--:-- --:--:-- --:--:--   371
=> "[\"logs\",\"pulpnode\",\"puppet\",\"puppetca\",\"templates\",\"tftp\"]"
```